### ISSUE TITLE
Ctrl-C can't stop windows processes

### ISSUE THREAD

**Issue by Chronial** [mintty/wsltty#332]
Steps to reproduce:
* Open a wsltty terminal
* Run `ping.exe localhost`
* Press ctrl-C

Expected outcome:
* Ping is aborted

Actual outcome:
* Nothing happens

wsltty version: 3.5.4
This happens with both wsl1 and wsl2
ctrl-c works with linux processes
ctrl-c works when running wsl inside the windows terminal.

**Comment by mintty** [comment]
If you use WSL ping rather than Windows ping.exe, it works.
Anyway, this is not a terminal issue. The terminal just passes ^C to the pty driver as a normal key. It's the driver, then cygwin, and respectively layers of WSL vs Windows that continue handling. Therefore closing.

**Comment by mintty** [comment]
It might be possible to handle this in wslbridge however, therefore reopening. @Biswa96?

**Comment by Biswa96** [comment]
If I run wslbridge2.exe directly in CMD, then Ctrl+C works with ping.exe.

**Comment by Chronial** [comment]
> Anyway, this is not a terminal issue. 

Wouldn't the fact that it works in windows terminal imply otherwise? Or are you saying this is a wslbridge issue?

> If I run wslbridge2.exe directly in CMD, then Ctrl+C works with ping.exe.

Same here

**Comment by Chronial** [comment]
Interestingly, running wslbridge in wsltty and ping.exe inside wslbridge makes Ctrl-C work again.
At same time, running tmux inside CMD and ping.exe in there breaks Ctrl-C.

## Doesn't Work
cmd → wsl → tmux → ping.exe
wsltty → ping.exe
wsltty → tmux → ping.exe

## Works
cmd → wsl → ping.exe
cmd → wslbridge ping.exe
wsltty → wslbridge ping.exe
wsltty → tmux → wslbridge ping.exe
wsltty → wslbridge → tmux → ping.exe

Long story short: Manually running wslbridge as the root process inside wsltty make everything work correctly. But I thought that's what wsltty was doing already? Something seems wrong here.

**Comment by mintty** [comment]
> Manually running wslbridge as the root process inside wsltty make everything work correctly.

Which means exactly what? Changed the shortcut?

> But I thought that's what wsltty was doing already? Something seems wrong here.

Yes, puzzled what you are doing.

**Comment by Chronial** [comment]
Start wsltty, run `ping.exe localhost`, press ctrl-C -> no effect.
Start wsltty, run wslbridge2.exe, run `ping.exe localhost`, press ctrl-C -> works

I am confused because wsltty seems to be running wslbridge already, so it's surprising to me that a second layer of wslbridge fixes the issue.
